core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -4164,16 +4164,13 @@ class ReplicaManagerTest {
         mock(classOf[FetchDataInfo])
       }).when(spyRLM).read(any())
 
-      // Get the current type=DelayedRemoteFetchMetrics,name=ExpiresPerSec metric value before fetching
-      val curExpiresPerSec = safeYammerMetricValue("type=DelayedRemoteFetchMetrics,name=ExpiresPerSec").asInstanceOf[Long]
```

Comment:
As we don't use value from yammer now, it seems the `metricsToBeDeletedInTheEnd` is useless.